htdocs/comm/action/card.php

```diff
@@ -2403,6 +2403,20 @@ function setdatefields()
 			print '</td></tr>';
 		}
 
+		//mail information
+		if (!empty($object->email_msgid) && $conf->global->COMM_ACTION_MAIL_DESCRIPTION) {
```

Comment:
All this information are short. So we can show them all the time when email_msgid is set.
No need for option inflation COMM_ACTION_MAIL_DESCRIPTION. You can just add the lines after the 
`if (!empty($object->email_msgid)`